src/dug/core/crawler.py

```diff
@@ -248,9 +249,12 @@ def expand_to_dug_element(self,
                         if target_node_type in node["category"]:
                             if node['id'].startswith(curie_filter):
                                 element_attribute_args = {"elem_id": node_id, "elem_type": dug_element_type}
-                                element_attribute_args.update({key: node.get(attribute_mapping[key], "")
-                                                               for key in attribute_mapping
-                                                               })
+                                for key in attribute_mapping:
+                                    mapped_value = node.get(attribute_mapping[key], "")
+                                    # treat all attributes as strings 
+                                    if key in array_to_string and isinstance(mapped_value, list) and len(mapped_value) > 0:
```

Comment:
This code suggests that a better name for `array_to_string` might be `list_field_choose_first` or something.